### ISSUE TITLE
fonts.gstatic.com blocked due gstaticadssl.l.google.com over cname

### ISSUE THREAD

**Issue by crssi** [lightswitch05/hosts#213]
Hi Daniel

Would you mind removing `gstaticadssl.l.google.com` from [ads-and-tracking-extended.txt](https://raw.githubusercontent.com/lightswitch05/hosts/master/docs/lists/ads-and-tracking-extended.txt), since the `fonts.gstatic.com` is a CNAME ([#1](https://mxtoolbox.com/SuperTool.aspx?action=a%3afonts.gstatic.com&run=toolpage), [#2](https://dnschecker.org/#CNAME/fonts.gstatic.com)) for it and it blocks google fonts now.

See more https://github.com/nextdns/metadata/issues/361#issuecomment-669123543

Thank you and cheers

**Comment by lightswitch05** [comment]
Please see #197 and #136

**Comment by lightswitch05** [comment]
I added a section to the README about this: https://github.com/lightswitch05/hosts#common-issues

**Comment by crssi** [comment]
Understand. Thank you for fast response and sorry for duplicate issue.

Cheers

**Comment by romaincointepas** [comment]
@lightswitch05 something important to consider:

Some major stub resolvers (macOS, iOS, unbound) chase CNAMEs (because they cache them separately, and for many other funky reasons), which means that a large percentage of users using your blocklist will have `fonts.gstatic.com` blocked even if they did not manually chose to apply it to CNAMEs (and even if they didn't enable deep CNAME inspection in Pihole, Adguard, or NextDNS).

Basically, QNAMEs-only blocklists don't really make sense. There are tons of domains not being blocked because they break functionality, IMO this should be treated the same way.
